chromium_src/chrome/browser/ui/views/frame/browser_view.cc

```diff
@@ -58,3 +62,20 @@ void BrowserView::SetNativeWindowPropertyForWidget(views::Widget* widget) {
 
   widget->SetNativeWindowProperty(kBrowserViewKey, this);
 }
+
+void BrowserView::UpdateExclusiveAccessBubble(
+    const ExclusiveAccessBubbleParams& params,
+    ExclusiveAccessBubbleHideCallback first_hide_callback) {
+  // Show/Hide full screen reminder bubble based on our settings preference
+  // for tab-initiated ones.
+  if (base::FeatureList::IsEnabled(features::kBraveShowFullscreenReminder)) {
+    if (!GetProfile()->GetPrefs()->GetBoolean(kEnableShowFullscreenReminder) &&
```

Comment:
Could probably be condensed into a single if statement.

Reply:
Done.